wsmaster/codenvy-hosted-integration-tests/codenvy-integration-tck-jpa/pom.xml

```diff
@@ -193,8 +192,7 @@
                                         che-core-api-user,
                                         che-core-api-machine,
                                         che-core-api-workspace,
-                                        che-core-api-ssh
-                                    </includeArtifactIds>
```

Comment:
I run mvn sortpom:sort command. It formatted a pom file on its own. Do you think we need to format code each time we run the sortpom?